utils/redis_init_script.tpl

```diff
@@ -15,8 +15,13 @@ case "$1" in
             echo "$PIDFILE does not exist, process is not running"
         else
             PID=$(cat $PIDFILE)
+            AUTH=$(grep ^requirepass $CONF | awk '{print $2}')
+            if [ "$AUTH" != "" ]
```

Comment:
Make it `$(awk '/^requirepass/ { print $2 }' $CONF)` and `[ -n "$AUTH" ]` and I'm happy.